plugins/apple/servicetalk/source.yaml

```diff
@@ -0,0 +1,4 @@
+source:
```

Comment:
Oh I sometimes forget about this file. I wonder if we should have a quick lint to fail if a directory doesn't contain this file.